wcc-contentful-app/app/models/wcc/contentful/app/custom_markdown_render.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module WCC::Contentful::App
+  class CustomMarkdownRender < Redcarpet::Render::HTML
+    def initialize(options)
+      super
+      @options = options
+    end
+
+    def link(link, title, content)
+      link_with_class_data =
+        @options[:links_with_classes].find do |link_with_class|
+          link_with_class[0] == link &&
+            link_with_class[2] == CGI.unescape_html(content)
+        end
+
+      link_class = link_with_class_data ? link_with_class_data[3] : nil
+      ActionController::Base.helpers.link_to(
+        content,
+        link,
+        hyperlink_attributes(title, link, link_class)
+      )
+    end
+
+    def hyperlink_attributes(title, url, link_class = nil)
+      default_link_attrs =
+        use_target_blank?(url) ? @options[:link_attributes] : @options[:link_attributes].except(:target)
```

Comment:
i think we should ensure `target: :_blank` is true here... other than that we're good to go